.travis.yml

```diff
@@ -2,6 +2,10 @@ language: minimal
 
 services: docker
 
+branches:
+  only:
+    - master
```

Comment:
guess we don't need something more elaborate like https://github.com/libero/browser/blob/master/.travis.yml#L35-L39